tests/integration/test_storage_s3/test.py

```diff
@@ -1021,14 +1028,24 @@ def test_truncate_table(started_cluster):
             len(list(minio.list_objects(started_cluster.minio_bucket, "truncate/")))
             == 0
         ):
-            return
```

Comment:
Here we had a bug in test. It just PASSED with this return and never checked what is read